src/main/java/org/jenkinsci/main/modules/sshd/IdleTimeout.java

```diff
@@ -43,13 +46,15 @@ public void apply(SshServer sshd) {
 			return;
 		}
 
-		sshd.getProperties().put(ServerFactoryManager.IDLE_TIMEOUT, timeoutInMilliseconds);
+		sshd.getProperties().put(CoreModuleProperties.IDLE_TIMEOUT.getName(), timeoutInMilliseconds);
 		// Read timeout must also be changed
 		long readTimeout = 0;
 		if (timeoutInMilliseconds != 0) {
-			readTimeout = ServerFactoryManager.DEFAULT_NIO2_READ_TIMEOUT - ServerFactoryManager.DEFAULT_IDLE_TIMEOUT + timeoutInMilliseconds;
+			readTimeout =
```

Comment:
NIO2_READ_TIMEOUT is now 0 by default so I dunno if this makes sense anymore, I think is not.